modules/serverless-logic-overview-parallelism.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+// * about/serverless-logic-overview.adoc
+
+
+:_content-type: CONCEPT
+[id="serverless-logic-overview-parallelism_{context}"]
+= Parallelism
+
+OpenShift Serverless Logic serializes the execution of parallel tasks. The word `parallel` does not indicate simultaneous execution, but it means that there is no logical dependency between the execution of branches. An inactive branch can start or resume the execution of a task without waiting for an active branch to be completed if the active branch suspends its execution. For example, an active branch may suspend its execution while waiting for an event reception.
+
+A parallel state is a state that splits up the current workflow instance execution path into multiple paths, one for each branch. These execution paths are performed in parallel and are joined back into the current execution path depending on the defined `completionType` parameter value.
```

Comment:
```suggestion
A parallel state is a state that splits up the current workflow instance execution path into multiple paths, one for each branch. These execution paths are performed in parallel and are joined back into the current execution path depending on the defined `completionType` parameter value.

`allOf`: All branches must complete execution before the state can transition/end. This is the default value in case this parameter is not set.

```